scripts/run_plugin_tests.py

```
@@ -0,0 +1,115 @@
+REPO_ROOT = Path(__file__).resolve().parents[1]
+PLUGINS = REPO_ROOT / "plugins"
+
+PASS, FAIL, SKIP = 0, 1, 2
+
+
+def run_one(script: Path, core: Path | None, timeout: int) -> tuple[int, str]:
+    env = dict(os.environ)
+    if core:
+        env["PYTHONPATH"] = f"{core}{os.pathsep}{env.get('PYTHONPATH', '')}"
+    try:
+        proc = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)